administration/urls.py

```diff
@@ -1,9 +1,11 @@
-from django.urls import path
-from .views import UsersListView, UserDetailView
+from django.urls import path, re_path, include
+from .views import AdminProfileList, AdminProfileDetail
```

Comment:
please add one pass line between libraries and custom modules

and please use one style of imports in whole project, here you have relative import, in previous file you have full path import

Reply:
Fixed